src/views/login/content/LoginCard.vue

```diff
@@ -45,11 +52,20 @@ export default class LoginCard extends Vue {
   }
 
   public sendForm (): void {
-    this.$refs.registerForm.validate((valid) => {
+    this.$refs.loginForm.validate((valid) => {
       if (valid) {
-        console.log(this.loginForm)
+        axios.post('/AuthenticateUser/login', this.loginForm).then(
+          (response) => {
+            console.log(`received token ${response.data.message}`)
+            this.setToken(response.data.message)
+            this.$router.push('/')
```

Comment:
Authentication is not properly implemented on the backend yet, so this section will be reworked when the backend is ready.